src/main/java/bio/terra/common/tracing/TracingProperties.java

```diff
@@ -1,50 +0,0 @@
-package bio.terra.common.tracing;
-
-import com.google.common.collect.Lists;
-import java.util.List;
-import org.springframework.boot.context.properties.ConfigurationProperties;
-
-/**
- * Defines configuration properties for the tracing package; see {@link TracingConfig} for their
- * use.
- */
-@ConfigurationProperties(prefix = "terra.common.tracing")
-public class TracingProperties {
-  /** Rate of trace sampling, 0.0 - 1.0 */
-  private double samplingRate = 0.05;
```

Comment:
This seems like a useful configuration option, why remove it?

Reply:
it is configured in open telemetry directly
https://github.com/DataBiosphere/terra-java-project-template/pull/21/files#diff-4dab07009b19ea187f607a407b00997345505227a852e7115e4dcd086f27a388R89